plugins/catalog-backend/src/service/DefaultEntitiesCatalog.ts

```diff
@@ -368,4 +372,27 @@ export class DefaultEntitiesCatalog implements EntitiesCatalog {
 
     return { facets };
   }
+
+  async changes(request: EntityChangesRequest): Promise<EntityChangesResponse> {
+    let query = this.database('final_entities')
+      .orderBy('change_index', 'asc')
+      .limit(MAX_CHANGES_PER_RESPONSE)
+      .select('entity_id', 'entity_ref', 'change_index', 'deleted_at');
```

Comment:
This can ofc forward anything we want from the `final_entites` table and we can introduce other forms of polling or streaming as well. More interested in feedback on the changes to the stitcher and deletion flow for now :grin: